script/Arcueid/Day 4/04_00_ARC04_6_2D.txt

```diff
@@ -46,12 +46,13 @@ My antibodies can render secret arts like Runes and Kabbalah useless, yet someho
 [sha:83a6fb4f1ea0647feda7120d38aef85214c6413c]{
 -- Page 566, Offset 7093.
 -- 答えて志貴。あなた、どんな年代物の神秘で、わたしを再起不能にしてくれたの？」
-Answer me, Shiki. What kind of %{i}Mystery%{/i} did you use to incapacitate me?"
+//could also use "vintage item" here instead of antique.
```

Comment:
how about relic?

Reply:
Feel like its maybe a bit too Christian of a nuance.

Thinking about it more I dunno if there are many other words than antique that fit